README.md

```diff
@@ -1,6 +1,6 @@
 # NI JavaScript and TypeScript Style Guide
 
-![Logo](https://raw.githubusercontent.com/ni/javascript-styleguide/HEAD/resources/logo.svg)
+![Logo](./resources/logo.svg)
```

Comment:
The githubusercontent path is so the logo is not broken on pages like the npm published page: https://www.npmjs.com/package/@ni/eslint-config

See: https://github.com/ni/javascript-styleguide/pull/6/files#r508710719

Can you restore the path and add a comment to the README ie
```
<!-- Preserve the raw.githubusercontent.com path to enable image rendering when viewed from https://www.npmjs.com/package/@ni/eslint-config -->
```

Reply:
Good suggestion. I ended up doing this in #21